rclip/model.py

```diff
@@ -37,14 +39,35 @@ def compute_text_features(self, text: List[str]) -> np.ndarray:
 
     return text_encoded.cpu().numpy()
 
+  # See: https://meta.wikimedia.org/wiki/User-Agent_policy
+  def download_image(self, url) -> Image.Image:
+    headers = {'User-agent':
+               "rclip - (https://github.com/yurijmikhalevich/rclip)"}
+    img = Image.open(requests.get(url, headers=headers, stream=True).raw)
```

Comment:
We need to be careful here not to try and download a huge file.

I think that checking Content-Length beforehand + adding a timeout should work well enough.

The limit of 50Mb will be enough for me. What do you think?